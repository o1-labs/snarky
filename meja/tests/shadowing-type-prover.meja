type t = A;

Prover {
  type t = B;
};
