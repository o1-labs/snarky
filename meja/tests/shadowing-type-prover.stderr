File "tests/shadowing-type-prover.meja", line 4, characters 7-8:
Error: There is already a type with name t.
